app/models/event1/MembersOrEmployersSummary.scala

```diff
@@ -37,27 +37,29 @@ object MembersOrEmployersSummary {
     (
       (JsPath \ "membersDetails" \ "firstName").readNullable[String] and
         (JsPath \ "membersDetails" \ "lastName").readNullable[String] and
-        readsMemberOrEmployerValue
+        readsMemberOrEmployerValue and
+        (JsPath \ "memberStatus").readNullable[String]
       ) (
-      (firstName, lastName, paymentValue) => {
+      (firstName, lastName, paymentValue, memberStatus) => {
         (firstName, lastName, paymentValue) match {
-          case (Some(fn), Some(ln), _) =>  MembersOrEmployersSummary(fn + " " + ln, paymentValue)
-          case (None, Some(ln), _) =>  MembersOrEmployersSummary(ln, paymentValue)
-          case (Some(fn), None, _) =>  MembersOrEmployersSummary(fn, paymentValue)
-          case (None, None, _) =>  MembersOrEmployersSummary(messages("site.notEntered"), paymentValue)
+          case (Some(fn), Some(ln), _) =>  MembersOrEmployersSummary(fn + " " + ln, paymentValue, memberStatus)
+          case (None, Some(ln), _) =>  MembersOrEmployersSummary(ln, paymentValue, memberStatus)
+          case (Some(fn), None, _) =>  MembersOrEmployersSummary(fn, paymentValue, memberStatus)
+          case (None, None, _) =>  MembersOrEmployersSummary(messages("site.notEntered"), paymentValue, memberStatus)
         }
       }
     )
 
   private def readsEmployerSummary(implicit messages: Messages): Reads[MembersOrEmployersSummary] =
     (
       (JsPath \ s"event${Event1.toString}" \ "companyDetails" \ "companyName").readNullable[String] and
-        readsMemberOrEmployerValue
+        readsMemberOrEmployerValue and
+        (JsPath \ "memberStatus").readNullable[String]
```

Comment:
I wonder whether we even need this. We could change it to a Reads[Option[..]] and read it as None if deleted then filter it out when used with a flatten/ flatmap. That way we avoid having to worry about deleted flags in the frontend.